modules/serverless-rn-1-10-1.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-10-1_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.10.1
+
+This release of {ServerlessProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+[id="fixed-issues-1-10-1_{context}"]
+== Fixed issues
+
+* The universal base images (UBIs) have been upgraded from `go-toolset-container-1.14.7-6` and `ubi8-minimal-container-8.2-349` in 1.10.0 to `go-toolset-container-1.14.12-11` and `ubi8-minimal-container-8.3-230` in this release.
```

Comment:
Maybe let's just mention the ubi8 image, not the go-toolset when we're talking about universal base images. Otherwise it looks good to me.